process/smartContract/process.go

```diff
@@ -1153,6 +1152,21 @@ func (sc *scProcessor) isCrossShardESDTTransfer(tx data.TransactionHandler) (str
 		return returnData, true
 	}
 
+	if function == vmcommon.BuiltInFunctionMultiESDTNFTTransfer {
```

Comment:
suggestion to have this functionality embedded in the parser directly?

Reply:
this is pretty specific. and used in one place only.